src/lib/bitcoin/BitcoinRequestParserMixin.js

```diff
@@ -0,0 +1,234 @@
+/* global Nimiq */
+/* global Errors */
+/* global BitcoinJS */
+/* global BitcoinUtils */
+
+/**
+ * @template {{}} T
+ * @typedef {new (...args: any[]) => T} Constructor;
+ */
```

Comment:
This method is taken from official TS docs: https://www.typescriptlang.org/docs/handbook/mixins.html